README.md

```diff
@@ -45,6 +45,16 @@ Mobile extensions don't have a Library Sandbox portion. Running `npx @adobe/reac
 
 You are welcome to commit `.sandbox` to your version control repository.
 
+### Building the Library
```

Comment:
Please add a note that this command does not work for mobile extensions and ensure it doesn't output anything for mobile

Reply:
Similar to `init` 

https://github.com/adobe/reactor-sandbox/pull/93/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R44